### ISSUE TITLE
Tracking execution in the Robin Editor

### ISSUE THREAD

**Issue by troybonneau** [robin-language/robin#1]
Right now debugging a process is a little hard to follow even when you're outputting information to the console. It would be nice to have the line of the current action be highlighted similar to how WinAutomation/ProcessRobot works.

**Comment by pagratios** [comment]
A full-fledged IDE will be available with the next generation of WinAutomation, which will be powered by Robin.

**Comment by jpapadimitriou** [comment]
I am closing this thread since this isn't an issue/bug.
